Moonscraper Chart Editor/Assets/Scripts/Game/Charts/IO/Chart/ChartWriter.cs

```diff
@@ -541,10 +541,15 @@ static void AppendNoteData(SongObject songObject, in SongObjectWriteParameters w
                     // Only need to get the flags of one note of a chord
                     if (note.next == null || (note.next != null && note.next.tick != note.tick))
                     {
-                        Note.Flags flagsToIgnore;
-                        if (!ChartIOHelper.c_drumNoteDefaultFlagsLookup.TryGetValue(note.rawNote, out flagsToIgnore))
+                        // Flag overrides
+                        foreach (var flags in ChartIOHelper.c_noteFlagOverrideLookup)
                         {
```

Comment:
I'm just thinking along the lines of "better safe than sorry" here, it would make more sense for the output to only be what should be respected rather than have potentially bad conflicts in places.
But yes, this should be handled by the flag toggles. I'll go ahead and make this editor-only.